src/pages/fr/reference/configuration-reference.md

```diff
@@ -0,0 +1,507 @@
+---
+# NOTE: This file is auto-generated from 'scripts/docgen.mjs'
+# Do not make edits to it directly, they will be overwritten.
+
+layout: ~/layouts/MainLayout.astro
+title: Référence de configuration
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+
+La référence suivante couvre toutes les options de configuration prises en charge dans Astro. Pour en savoir plus sur la configuration d'Astro, lisez notre guide sur la [configuration d'Astro](/fr/guides/configuring-astro/).
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config'
+
+export default defineConfig({
+  // Vos options de configuration ici...
+})
+```
+## Options de niveau supérieur
+
+### racine
```

Comment:
This heading refers to the name of the config option, so we should leave it in English.

```suggestion
### root
```

Reply:
I just have on more problem. I can't translate Legacy Flags correctly. So I think the associated paragraph may have small translation problems.